README.md

```diff
@@ -641,11 +641,10 @@ er:
   # Hide relation definition from ER diagram
   # Default is false
   hideDef: true
-  # Hide column settings in ER diagram
-  hideColumnTypes:
-    # Hide not related columns from ER diagram
-    # Default is false
-    notRelated: true
+  # Show column settings in ER diagram
+  showColumnTypes:
```

Comment:
I thought it would be kinder to mention in the README that all columns will be displayed if this setting is not specified.

Reply:
Yes